### DISCUSSION TITLE
Exclue entries of a feed per pubdate

### DISCUSSION THREAD

**Discussion by update-freak** [FreshRSS/FreshRSS#6153] [Q&A] [answered]
Hi everyone,

I would like to create an RSS feed of "tagesschau in 100 Sekunden" that I got one video update per day.
But with the search https://mediathekviewweb.de/#query=tagesschau%20in%20100%20Sekunden (equal RSS feed: https://mediathekviewweb.de/feed?query=tagesschau%20in%20100%20Sekunden) there are multiple items at different times per day -> but I would like to have only one entry and not e.g. 10 a day, which "fills up" my RSS feed overview.

Is it somehow possible that I can define that it takes the last "Tagesschau in 100 Sekunden" before 10 pm once a day? --> best that I then only receive the most current (latest video) before 10pm or that every item is directly mark as read and I only the latest one before 10pm is not marked as read.

I'm not sure if this is possible via filter, because I want to use it on my smartphone with a RSS client (e.g. FeedMe or Fluent Reader).
As I unterstood the filter option only creates an view but do not exclude entries.

**Comment by Alkarex** [comment, marked as answer]
You could use the option to keep only the newest article as unread.
Otherwise, a scenario like that would require that you write a little FreshRSS extension

**Reply by update-freak** [reply]
ok, thank you very much.
